experimental/rocm/CMakeLists.txt

```diff
@@ -12,16 +12,6 @@ cmake_path(ABSOLUTE_PATH IREE_PACKAGE_ROOT_DIR
   OUTPUT_VARIABLE IREE_PACKAGE_ROOT_DIR)
 set(IREE_PACKAGE_ROOT_PREFIX iree)
 
-set(IREE_ROCM_TARGET_CHIP_DEFAULT "gfx908")
-set(IREE_ROCM_TARGET_CHIP "${IREE_ROCM_TARGET_CHIP_DEFAULT}" CACHE STRING
-  "Target chip for ROCm. This influences conformance tests that need to compile device code. Defaults to \"${IREE_ROCM_TARGET_CHIP_DEFAULT}\".")
-set(IREE_ROCM_LINK_BC_DEFAULT OFF)
-set(IREE_ROCM_LINK_BC ${IREE_ROCM_LINK_BC_DEFAULT} CACHE BOOL
-  "Whether to try Linking to AMD Bitcodes. This influences conformance tests that need to compile device code. Defaults to ${IREE_ROCM_LINK_BC}.")
-set(IREE_ROCM_BC_DIR_DEFAULT "/opt/rocm/amdgcn/bitcode")
-set(IREE_ROCM_BC_DIR "${IREE_ROCM_BC_DIR_DEFAULT}" CACHE STRING
-  "Directory of ROCM Bitcode. This influences conformance tests that need to compile device code. Defaults to \"${IREE_ROCM_TARGET_CHIP_DEFAULT}\".")
```

Comment:
Relating to my comment above... should this stay here, and then HIP could use its own options?

How long will `experimental/rocm` remain? Does HIP fully replace it?

Reply:
Changed to not touching `experimental/rocm` anymore. HIP should fully replace it.